src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -298,6 +286,7 @@ ProgressiveProfiling.propTypes = {
 };
 
 ProgressiveProfiling.defaultProps = {
+  authenticatedUser: {},
```

Comment:
is should be required, no?

Reply:
No, for embedded experience we use the getAuthenticatedUser() to get the details.